create-labels.py

```diff
@@ -0,0 +1,100 @@
+# This script removes default github labels and adds custom ones using python requests and the github API
+# To work it needs: 1) User token, 2) Target username / organization name and target repository name and the 
+# By Pablo and Spearbit with <3
+# License: MIT
+
+import requests
+
+
+def delete_default_labels():
+    default_labels = [
+        "bug", 
+        "duplicate",
+        "enhancement", 
+        "invalid", 
+        "question", 
+        "wontfix", 
+        "question", 
+        "good first issue", 
+        "help wanted", 
+        "documentation"
+        ]
+    for i in default_labels:
+        r = requests.delete(f"https://api.github.com/repos/{username}/{repo}/labels/{i}", headers=headers)
+        print(r)
+    print("Done deleting default labels!\n")
+
+
+def create_new_labels():
+    new_labels = [
+        "Severity: Critical Risk", 
+        "Severity: High Risk",
+        "Severity: Medium Risk", 
+        "Severity: Low Risk", 
+        "Severity: Informational",
+        "Severity: Gas Optimization",
+        "Status: Acknowledged",
+        "Status: Fixed",
+        "Status: ReadyForReport"
+    ]
+
+    colors = [
+        "ff0000",
+        "B60205",
+        "D93F0B",
+        "FBCA04",
+        "1D76DB",
+        "B4E197",
+        "5319E7",
+        "0E8A16",
+        "bfdadc"
+    ]

```

Comment:
I'd probably use a dictionary.  Something like
```python
labels = [
    {'severity': 'Severity: Critical Risk', 'color': 'ff00000'},
    {'severity': ...}
```

Reply:
Makes sense, i'll address it